project/cmake/addons/addons/pvr.filmon/pvr.filmon.txt

```diff
@@ -1 +1 @@
-pvr.filmon https://github.com/kodi-pvr/pvr.filmon a0ee270
+pvr.filmon https://github.com/kodi-pvr/pvr.filmon 14419e3
```

Comment:
Can you swap this to 0e5bad6ae0.

The fix for windows is merged, see https://github.com/kodi-pvr/pvr.filmon/pull/5.